src/DDAModel.cpp

```diff
@@ -1311,6 +1311,11 @@ VectorXd DDAModel::get_parameter_copy( ) {
     return parameters;
 }
 
+// Making this return a copy, temporarily, to get python bindings to work.
```

Comment:
Again, not sure if we need to have this? Happy to include it but only if there is a good reason for the API method near-duplication with get_parameters.

Reply:
we will not need it, but I have not gone around to making this change yet. I will need to be careful with using symmetries / extrusions to go from the 2D symmetric to the full 3D, and I do not have the time to do this right now. Since the c++ side provides it, I am using it temporarily.

Reply:
SG. If I were being pedantic, I would ask to rename the `getDielectrics` method to `_getDielectrics`. The leading underscore is often used to indicate that we shouldn't be relying on / calling this function, since it might change or get deprecated in the future. It's like a flag that says "this code is slightly uglier than normal because we need to clean it up - it also might cause bugs once we deprecate."

Reply:
we need to remove this anyways so i will leave it as is until we get to it.